airflow/utils/db.py

```diff
@@ -1331,9 +1333,9 @@ def _move_duplicate_data_to_new_table(
     """
     bind = session.get_bind()
     dialect_name = bind.dialect.name
+
     query = (
-        session.query(source_table)
-        .with_entities(*[getattr(source_table.c, x.name).label(str(x.name)) for x in source_table.columns])
+        session.select(*[getattr(source_table.c, x.name).label(str(x.name)) for x in source_table.columns])
```

Comment:
There's no `session.select`. This will fail

Reply:
wonder why no tests failed. corrected it

Reply:
Coverage in this file is not great because most of these handle database upgrades and can’t be very efficiently tested.